src/components/Search/SearchResults/SearchResults.tsx

```diff
@@ -7,30 +7,35 @@ export interface SearchResultsProps extends React.HTMLAttributes<HTMLElement> {
   isLoading?: boolean;
```

Comment:
Can we do this so the results interface is sharable with consumers and then reuse this interface in Search.stories.

```
export interface SearchResult {
  id: string;
  url: string;
  label: string;
}

...
autoCompleteResults?: SearchResult;
```